code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -323,8 +423,61 @@
 	cardigan["cardigan"] = /obj/item/clothing/suit/storage/toggle/cardigan
 	cardigan["sweater cardigan"] = /obj/item/clothing/suit/storage/toggle/cardigan/sweater
 	cardigan["argyle cardigan"] = /obj/item/clothing/suit/storage/toggle/cardigan/argyle
+	cardigan["crop top cardigan"] = /obj/item/clothing/suit/storage/toggle/cardigan/croptop
 	gear_tweaks += new/datum/gear_tweak/path(cardigan)
 
+/datum/gear/suit/denim_jacket
+	display_name = "denim jacket (selection)"
+	description = "A selection of denim jackets."
+	path = /obj/item/clothing/suit/storage/toggle/denim_jacket
+
+/datum/gear/suit/denim_jacket/New()
+	..()
+	var/denimjackets = list()
+	denimjackets["denim jacket"] = /obj/item/clothing/suit/storage/toggle/denim_jacket
+	denimjackets["denim jacket, corporate"] = /obj/item/clothing/suit/storage/toggle/denim_jacket/nanotrasen
+	denimjackets["denim vest"] = /obj/item/clothing/suit/storage/toggle/denim_jacket/sleeveless
+	denimjackets["denim vest, corporate"] = /obj/item/clothing/suit/storage/toggle/denim_jacket/nanotrasen/sleeveless
+	gear_tweaks += new/datum/gear_tweak/path(denimjackets)
+
+/datum/gear/suit/greatcoat
+	display_name = "greatcoat"
```

Comment:
Remove the greatcoat. They also have the same issue of cloaks.

Reply:
For reference:
<details>
<summary>Click:</summary>
<img src="https://user-images.githubusercontent.com/3950115/85944047-de511280-b8e8-11ea-937d-3b121ad9ba48.png">
</details>